iio-widgets/src/datastrategy/channelattrdatastrategy.cpp

```diff
@@ -0,0 +1,61 @@
+#include "datastrategy/channelattrdatastrategy.h"
+#include <QFile>
+
+#define BUFFER_SIZE 256
+
+Q_LOGGING_CATEGORY(CAT_ATTR_DATA_STRATEGY, "AttrDataStrategy")
+
+using namespace scopy::attr;
```

Comment:
Is there a need to add an extra namespace ? I would just keep it under 
namespace scopy { }